docs/05-technical-reference/00-architecture/evnt-01-architecture.md

```diff
@@ -2,30 +2,32 @@
 
-## Event Publisher Proxy
+## Event processing and delivery
 
-Event Publisher Proxy receives legacy and Cloud Event publishing requests from the cluster workloads (microservices or Functions). It converts any legacy events to Cloud Events. Then, it redirects events to the NATS server. It also fetches a list of Subscriptions for a connected application.
+The event processing and delivery flow uses the NATS server to process events and send them to subscribers.
+This diagram explains the event flow in Kyma, from the moment the Application sends an event, to the point when the event triggers the Function.
 
-## Eventing Controller
+![Eventing flow](./assets/evnt-architecture.svg)
+
+1. The Eventing Controller watches the Subscription Custom Resource. It determines if there are any new incoming events.
+
+2. The Eventing Controller creates an infrastructure for the NATS server.
 
-Eventing Controller manages the internal infrastructure in order to receive an event. It watches Subscription custom resource definitions. When an event is received in an Application, it lays down the Eventing infrastructure in NATS in order to trigger a Function. Eventing Controller also dispatches messages to subscribers such as a Function or another workload.
+3. The [Application Connector](../../01-overview/main-areas/application-connectivity/ac-01-application-connector.md) publishes events to the Event Publisher Proxy.
 
-## Event types
+4. The Event Publisher Proxy sends events to the NATS server.
 
-Eventing supports both Cloud Events and legacy events. Event Publisher Proxy converts legacy events to Cloud Events and adds the `sap.kyma.custom` prefix.
+5. The NATS server dispatches events to the Eventing Controller.
 
-For a Subscription Custom Resource, the fully qualified event type takes the sample form of `sap.kyma.custom.commerce.order.created.v1` or `sap.kyma.custom.commerce.Account.Root.Created.v1`.
+6. The Eventing Controller dispatches events to subscribers (microservices or Functions).
 
-The event type is composed of the following components:
-- Prefix: `sap.kyma.custom`
-- Application: `commerce`
-- Event: can have two or more segments separated by `.`; for example, `order.created` or `Account.Root.Created`
-- Version: `v1`
 
-For publishers, the event type takes this sample form:
-- `order.created` or `Account.Root.Created` for legacy events coming from the `commerce` application
-- `sap.kyma.custom.commerce.order.created.v1` or `sap.kyma.custom.commerce.AccountRoot.Created.v1` for Cloud Events
+## Event Publisher Proxy
+
+Event Publisher Proxy receives legacy and Cloud Event publishing requests from the cluster workloads (microservices or Functions). It converts any legacy events to Cloud Events. Then, it redirects events to the NATS server. It also fetches a list of Subscriptions for a connected application.
```

Comment:
Not sure - should "application" be capitalized? In all previous and following occurrences, it is.